Jenkinsfile

```diff
@@ -228,19 +230,23 @@ pipeline {
     }
     success {
       script {
-        if (!nxUtils.isPullRequest() && !isTriggeredByNuxeoPR()) {
-          if (!hudson.model.Result.SUCCESS.toString().equals(currentBuild.getPreviousBuild()?.getResult())) {
-            def triggeredBy = isTriggered() ? ", triggered by ${upstreamJobName} #${upstreamBuildNumber}" : ''
-            nxSlack.success(message: "Successfully built nuxeo/rest-api-compatibility-tests ${BRANCH_NAME} #${BUILD_NUMBER}${triggeredBy}: ${BUILD_URL}")
-          }
+        if (!nxUtils.isPullRequest() && !isTriggeredByNuxeoPR()
+            && nxUtils.previousBuildStatusIs(status: Result.FAILURE, ignoredStatuses: [Result.ABORTED, Result.NOT_BUILT])) {
+          nxTeams.success(
+            message: "Successfully built ${currentBuild.fullProjectName}",
```

Comment:
```suggestion
            message: "Successfully built ${currentBuild.fullProjectName} ${BRANCH_NAME}${triggeredBy}",
```

Reply:
I choosed to remove the `BRANCH_NAME` and the `triggeredBy` variables because they are set in the job description and so they will appear in the notification subtitle, should we also add it to the message?

Reply:
No, I didn't see this :+1: